backend/hdfs/hdfs.go

```diff
@@ -32,6 +32,25 @@ func init() {
 				Value: "root",
 				Help:  "Connect to hdfs as root",
 			}},
+		}, {
+			Name:     "service_principal_name",
+			Help:     "Enables KERBEROS authentication. Specifies the Service Principle Name (<SERVICE>/<FQDN>) for the namenode.",
```

Comment:
Note that we try to wrap the help at 80 characters. The first line becomes the command line help for the flag.

You can use the backquote character for writing strings over multiple lines, so I'd write


```go
Help: `Kerberos service principal name for the namenode.

Enables KERBEROS authentication. Specifies the Service Principal Name
(<SERVICE>/<FQDN>) for the namenode.
```